engine/src/test/scala/org/scalarules/dsl/nl/grammar/AfrondingsTestBerekening.scala

```diff
@@ -0,0 +1,37 @@
+package org.scalarules.dsl.nl.grammar
+
+import org.scalarules.dsl.nl.grammar.AfrondingsWords._
+import org.scalarules.dsl.nl.grammar.AfrondingsTestBerekeningGlossary._
+
+import scala.language.postfixOps
+
+class AfrondingsTestBerekening extends Berekening (
+
+  Gegeven(afrondingsType is "halfNaarEven") Bereken
```

Comment:
If you want to eliminate a `Fact`, you can actually have all these derivations use `Gegeven (altijd)`. Your tests only extract the useful value anyways.